File: 0268-missing-number/0268-missing-number.java
class Solution {
    public int missingNumber(int[] nums) {
        int sum = 0;
        for(int i = 0; i < nums.length; i++){
            sum = sum + nums[i];
        }
        int actualSum = (nums.length * (nums.length + 1)) / 2;
        int missingNum = actualSum - sum;

        return missingNum;
    }
}

File: 0268-missing-number/0268-missing-number.py
class Solution:
    def missingNumber(self, nums: List[int]) -> int:
        n = len(nums)
        sum1 = (n * (n + 1)) // 2 
        sum2 = sum(nums)
        missingNum = sum1 - sum2
        return missingNum
         
        